common/djangoapps/entitlements/api/v1/tests/test_views.py

```diff
@@ -7,6 +7,9 @@
 import pytz
 from django.conf import settings
 from django.core.urlresolvers import reverse
+
+from course_modes.models import CourseMode
```

Comment:
Check your import order, this seems slightly off to me. It might `from mock import patch` that is in the wrong place though.

Reply:
You can also run the tool to do the sort for you.  I believe it is called isort.  I need to look it up though.